ui/app/AppLayouts/Profile/views/ContactsView.qml

```diff
@@ -20,7 +20,7 @@ import "../popups"
 
 SettingsContentBase {
     id: root
-    onWidthChanged: { contentItem.width = width; }
+    onWidthChanged: { contentItem.width = contentWidth; }
     onHeightChanged: { contentItem.height = height; }
```

Comment:
Minor suggestion: it could be moved directly to `contentItem`:
```
        width: root.contentWidth
        height: root.height
```

Reply:
Yeah, there was a much bigger change done by @alexandraB99 in 1cda32470bf62af3218f44b1424b7b4ae9520645 which I didn't want to break :)